src/pages/Portfolio/MyKiva/MyKivaPage.vue

```diff
@@ -133,6 +142,30 @@ const handleSelectedLoan = loan => {
 	fetchLoanUpdates(activeLoan.value.id);
 };
 
+const fetchUserAchievements = () => {
+	apollo.query({ query: userAchievementProgressQuery })
+		.then(result => {
+			userAchievements.value = result.data?.userAchievementProgress?.tieredLendingAchievements ?? [];
+		}).catch(e => {
+			logReadQueryError(e, 'MyKivaPage userAchievementProgressQuery');
+		});
+};
+
+const fetchBadgesData = () => {
+	apollo.query({
+		query: contentfulEntriesQuery,
+		variables: {
+			contentType: 'challenge',
+			limit: 100,
```

Comment:
How many badges do we currently have? I wonder when this will not be enough...

Reply:
5 leves per badge, having 5 badges there are 25.
But as this a query for the whole challege set, we have `"total": 45`

We can change this to 200 to extra ensure.

Reply:
200 should be fine for now.